crates/api_models/src/payouts.rs

```diff
@@ -535,6 +535,16 @@ pub struct PayoutCreateResponse {
     /// Customer's phone country code. _Deprecated: Use customer object instead._
     #[schema(deprecated, max_length = 255, example = "+1")]
     pub phone_country_code: Option<String>,
+
+    /// (This field is not live yet)
+    /// Error code unified across the connectors is received here in case of errors while calling the underlying connector
+    #[remove_in(PayoutCreateResponse)]
+    pub unified_code: Option<String>,
```

Comment:
In  create api if somethings fails we throws 4xx so this response we will send only in valid case so these fields not required in create response